api/swagger-spec/api.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Looking at the script, I don't see where this file is generated from and it looks out of date.  Extraneous?